crates/common/src/compile.rs

```diff
@@ -496,26 +504,6 @@ pub fn compile_target<C: Compiler>(
     ProjectCompiler::new().quiet(quiet).files([target_path.into()]).compile(project)
 }
 
-/// Compiles an Etherscan source from metadata by creating a project.
-/// Returns the artifact_id, the file_id, and the bytecode
-pub async fn compile_from_source(
```

Comment:
why was this inlined?

Reply:
it's pretty small, only used in a single place and we need it to return 3 variables so it seemed better that way